safe_web_api_playground/package.json

```diff
@@ -21,20 +21,19 @@
   "dependencies": {
     "babelify": "^7.3.0",
     "body-parser": "^1.17.2",
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^4.3.1",
     "browserify": "^14.4.0",
     "codemirror": "^5.33.0",
     "envfile": "^2.1.1",
     "express": "^4.15.3",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.0",
```

Comment:
The major version change breaks `gulp.start` operations in `gupfile.js`. Needs updating.